### PR TITLE
Add Syself to ADOPTERS.md

### PR DESCRIPTION
**Pull request by janiskemper**
Adding Syself as adopter. Thanks a lot for the cool project!

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -40,3 +40,4 @@ This list is sorted in chronological order, based on the submission date.
...
 | [IBM](https://www.ibm.com) | @pgodowski | 2024-02-20 | IBM uses CloudNativePG as the embedded SQL database within the family of [IBM Cloud Pak](https://www.ibm.com/cloud-paks) products, running as customer-managed software on top of [OpenShift Container Platform](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform). |
 | [Google Cloud](https://cloud.google.com/) | @mastersingh24 | 2024-03-12 | Leverage the full potential of cutting-edge PostgreSQL  and CloudNativePG  on [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) with EDB Community 360 PostgreSQL available in the [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/product/public-edb-ppas/edb-postgresql). |
+| [Syself](https://syself.com) | @batistein | 2024-05-06 | Syself offers a simplified, multi cloud Managed Kubernetes platform based on Cluster Api and uses CloudNativePG for managing Postgres clusters in our internal infrastructure. |
```
Comment (gbartolini) [inline ADOPTERS.md:43]:
```suggestion
| [Syself](https://syself.com) | @batistein | 2024-05-06 | Syself offers a simplified, multi-cloud Managed Kubernetes platform based on Cluster API and uses CloudNativePG for managing Postgres clusters in our internal infrastructure. |
```

**Review by gbartolini — APPROVED** [review]

**Comment by gbartolini** [comment]
@janiskemper can you please rebase your patch?

File: ADOPTERS.md (hunk 1 above)
Comment (janiskemper) [reply ADOPTERS.md]: thanks, I was a bit sloppy!

**Comment by gbartolini** [comment]
@sxd just pointed out to me that you have added @batistein as the user requesting this. I am OK to add him, but could you please ensure that at least he adds a comment to this PR saying "I agree" or something? Thanks.

**Comment by batistein** [comment]
Hi @gbartolini thanks for the verification. Yes I agree!

**Comment by gbartolini** [comment]
Thank you. @janiskemper can you please proceed again with the rebase? Thanks!

**Comment by gbartolini** [comment]
Too late ... If you can do it now, great. Otherwise, let me ping you after we have released 1.23.2 - this is becoming a never ending story

**Review by mnencia — APPROVED** [review]

**Comment by janiskemper** [comment]
Thanks! Seems to have worked anyway somehow.

**Pull request merged**